ebmbot/slack.py

```diff
@@ -1,8 +1,13 @@
 from time import sleep
 
+from workspace.utils.blocks import get_basic_header_and_text_blocks
+
 from .logger import logger
 
 
+COULD_NOT_NOTIFY_SLACK = "Could not notify slack"
```

Comment:
I would be inclined to keep this as a variable within the functions rather than making it a global; it'll be minimal repetition (you can define it once and pass it to the `get_slack_error_blocks` function).

Reply:
Done! I kept the variable name unchanged (COULD_NOT_NOTIFY_SLACK) since I can't think of a better name.